homeassistant/components/growatt_server/sensor_types/mix.py

```diff
@@ -224,6 +224,7 @@
         api_key="etouser",
         native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
         device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
```

Comment:
What's the difference between this sensor and the other today energy sensors that don't have a state class?

Reply:
I'll remove (and the new drop threshold value) and re-add in a subsequent PR